web/src/reducers/index.js

```diff
@@ -23,6 +23,7 @@ const initialApplicationState = {
   legendVisible: false,
   locale: window.locale,
   onboardingSeen: Cookies.get('onboardingSeen') === 'true' || false,
+  openTooltipMode: null,
```

Comment:
Can we rename to something a bit more explicit? Open is a verb and therefore it's a bit hard to understand it as a state. Maybe just `tooltipDisplayMode`? Adding the word `display` makes it a bit more explicit that a `null` value should hide it.